src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__image">
+      </div>
+
+      <h4 class="card__productname">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h4>
+
+      <p class="card__productcode">Код товара: 195434</p>
+
+      <div class="review">
+        <div class="stars stars--1">
```

Comment:
```suggestion
        <div class="stars stars--4">
```
Число - это количество активных звездочек, у тебя их 4